test_libs/pyspec/eth2spec/test/block_processing/test_process_early_derived_secret_reveal.py

```diff
@@ -0,0 +1,130 @@
+import pytest
```

Comment:
Even though we have these nice decorators for specify which specs to run against, I'm still kind of in favor of having phase 1 tests in a separate directory